### PR TITLE
[PW-4392] Update README based on the new template

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,87 @@
...
-```python
+Use pip command: 
+~~~~ bash 
+pip install Adyen
+~~~~
+
+## Using the library
+ 
+ 
+### General use with basic auth
```
Comment (Aleffio) [inline README.md:45]: should we show here the use with the API key as a default?

Comment (AlexandrosMor) [reply README.md:45]: I used the existed example, I can add the example from e2e

File: README.md [hunk 2]
```diff
@@ -1,68 +1,86 @@
...
-* checkout
-* checkout utility
-* payments
-* modifications
-* payouts
-* recurring
-
-## Requirements
-
+This is the officially supported python library for using Adyen's APIs.
```
Comment (acampos1916) [inline README.md:4]:
```suggestion
This is the officially supported Python library for using Adyen's APIs.
```

File: README.md [hunk 3]
```diff
@@ -1,68 +1,86 @@
...
-```
-
-## Documentation
-* https://docs.adyen.com/developers/development-resources/libraries
-* https://docs.adyen.com/developers/checkout
+~~~~
+ 
+### Example integration
+ 
+For a closer look at how our python library works, clone our [example integration](https://github.com/adyen-examples/adyen-python-online-payments). This includes commented code, highlighting key features and concepts, and examples of API calls that can be made using the library.
```
Comment (acampos1916) [inline README.md:61]:
```suggestion
For a closer look at how our Python library works, clone our [example integration](https://github.com/adyen-examples/adyen-python-online-payments). This includes commented code, highlighting key features and concepts, and examples of API calls that can be made using the library.
```

File: README.md [hunk 4]
```diff
@@ -1,68 +1,86 @@
...
-
+ 
+We encourage you to contribute to this repository, so everyone can benefit from new features, bug fixes, and any other improvements.
+ 
+ 
+Have a look at our [contributing guidelines](https://github.com/Adyen/adyen-python-api-library/blob/develop/CONTRIBUTING.md) to find out how to raise a pull request.
+ 
+ 
+## Support
+If you have a feature request, or spotted a bug or a technical problem, [create an issue here](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2FAdyen%2Fadyen-web%2Fissues%2Fnew%2Fchoose).
```
Comment (acampos1916) [inline README.md:73]: Is this the correct link? I think this is a login + redirect , but the issue form alone should already do the login check

Comment (AlexandrosMor) [inline README.md:73]:
```suggestion
If you have a feature request, or spotted a bug or a technical problem, [create an issue here](https://github.com/Adyen/adyen-web/issues/new/choose).
```

**Review by acampos1916 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,68 +1,85 @@
...
+## Integration 
+The library supports all APIs under the following services:
+ 
+* [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/overview): Our latest integration for accepting online payments. Current supported version:  **v67**
+* [Payments API](https://docs.adyen.com/api-explorer/#/Payment/v64/overview): Our classic integration for online payments. Current supported version:  **v64**
+* [Recurring API](https://docs.adyen.com/api-explorer/#/Recurring/v49/overview): Endpoints for managing saved payment details. Current supported version:  **v49**
+* [Payouts API](https://docs.adyen.com/api-explorer/#/Payout/v64/overview): Endpoints for sending funds to your customers. Current supported version:  **v64**
+* [Orders API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/orders): Endpoints for creating and canceling orders. Current supported version:  **v67**
+* [Utility API](https://docs.adyen.com/api-explorer/#/CheckoutService/v1/post/originKeys): This operation takes the origin domains and returns a JSON object containing the corresponding origin keys for the domains. Current supported version:  **v1**
```
Comment (Aleffio) [inline README.md:13]: docs link needs to be updated to v67 as v1 is probably deprecated(endpoint should still work, but let's change it in another story?)
```suggestion
* [Utility API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/originKeys): This operation takes the origin domains and returns a JSON object containing the corresponding origin keys for the domains. Current supported version:  **v67**
```

**Review by Aleffio — APPROVED** [review]

**Review by acampos1916 — APPROVED** [review]

**Pull request merged**